### PR TITLE
doc: add Hugo-powered website

### PR DESCRIPTION
**Pull request by cyphar**
Hosted on https://umo.ci/

Signed-off-by: Aleksa Sarai <asarai@suse.de>

### CODE REVIEW

**Comment by cyphar** [comment]
LGTM.

**Comment by cyphar** [comment]
LGTM.

**Comment by cyphar** [comment]
LGTM.

File: doc/site/advanced/workflow-optimisation.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
++++
+title = "Workflow Optimistaion"
```
Comment (jonboulle) [inline doc/site/advanced/workflow-optimisation.md]: Optimisation

File: doc/site/advanced/workflow-optimisation.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+weight = 10
++++
+
+One of the first things that a user of umoci may notice is that certain
+operations can be quite expensive. Most notable, unpack and repack operations
```
Comment (jonboulle) [inline doc/site/advanced/workflow-optimisation.md]: notably

File: doc/site/advanced/workflow-optimisation.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+```text
+% umoci unpack --image image_build_XYZ:wip bundle_a
+% ./some_build_process_1 ./bundle_a
+% ./some_build_process_2 ./bundle_a
+% ./some_build_process_3 ./bundle_a
+% umoci repack --image image_build_XYZ:wip bundle_a
+% umoci tag --image image_build_XYZ:wip final
+```
+
+Do not dispair however, there is a flag just for you! With `--refresh-bundle`
```
Comment (jonboulle) [inline doc/site/advanced/workflow-optimisation.md]: despair

File: doc/site/advanced/workflow-optimisation.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+it is possible to preform the above operations without needing to do any extra
```
Comment (jonboulle) [inline doc/site/advanced/workflow-optimisation.md]: perform

File: doc/site/quick-start/creating-an-image.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+weight = 30
++++
+
+Creating a new image with umoci is fairly simple, and effectively involves
+creating an image "husk" which you can then [operate on][workflow] as though it
+was a normal image. New images contain no layers, and have a dummy
+configuration that should be replaced by a user.
+
+If you wish to create a new image layout (which contains nothing except the
+bare minimum to specify that the image is an OCI image), you can do so `umoci
```
Comment (jonboulle) [inline doc/site/quick-start/creating-an-image.md]: can do so with

**Comment by cyphar** [comment]
LGTM.

**Pull request merged**